### PR TITLE
S4 products list

### PR DESCRIPTION
**Pull request by lingsat**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL = 'https://645b959199b618d5f31f7299.mockapi.io'
```
Comment (Yuliia143) [inline .env]: Need empty line

File: src/features/products/components/ListSort/ListSort.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, { FC, useState } from "react";
+import arrowIcon from "@/assets/images/arrow_black.svg";
+import "./ListSort.scss";
+
+const ListSort: FC = () => {
```
Comment (Yuliia143) [inline src/features/products/components/ListSort/ListSort.tsx]: It isn't relevant to this task

**Review by Yuliia143 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (lingsat) [reply .env]: Fixed - add empty line

File: src/features/products/components/ListSort/ListSort.tsx (hunk 2 above)
Comment (lingsat) [reply src/features/products/components/ListSort/ListSort.tsx]: Fixed - remove this component

**Review by Yuliia143 — APPROVED** [review]

**Pull request merged**